src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -110,15 +112,26 @@ export class MeetingManager implements AudioVideoObserver {
 
   simulcastEnabled: boolean = false;
 
-  constructor(config: ManagerConfig) {
-    this.logLevel = config.logLevel;
+  videoDownlinkBandwidthPolicy: VideoDownlinkBandwidthPolicy | undefined;
+
+  logger: Logger | undefined;
```

Comment:
Change `null` to `undefined`

Reply:
Can't we just keep it as logger: Logger? No need to do undefined here.

Reply:
Actually due to the `"strictNullChecks": true` setting in the React SDK's tsconfig, if we do this then we cannot reset it to `undefined` in initialize function. In the JS SDK we do not have this config.

Reply:
Yeah Sichao also told me that. Thus, I have approved the PR :D